### PR TITLE
Update trader files for Streatman's new L85 and SVD models

### PR DESCRIPTION
**Pull request by worldwidesorrow**

### CODE REVIEW

**Comment by worldwidesorrow** [comment]
Added the weights for the rifles.

Removed BTR40_MG_TK_INS_EP1 from my prior update because it is not a DZE model like the rest of the vehicles. It will spawn with ammo which is against the DayZ Epoch armed vehicle idea.

File: SQF/dayz_code/Configs/CfgServerTrader/Category/FriendlyAssaultRifle.hpp [hunk 1]
```diff
@@ -479,17 +479,133 @@ class Category_485 {
...
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {12,"ItemGoldBar"};
+	};
+	class BAF_L85A2_RIS_SUSAT { //Not sure if this should be kept in 1.0.6.2.
 		type = "trade_weapons";
 		buy[] = {6,"ItemGoldBar"};
 		sell[] = {3,"ItemGoldBar"};
 	};
-	class BAF_L85A2_RIS_ACOG {
+	class BAF_L85A2_RIS_ACOG { //Streatman's model L85A2_ACOG_DZ should replace this.
```
Comment (ebayShopper) [inline SQF/dayz_code/Configs/CfgServerTrader/Category/FriendlyAssaultRifle.hpp]: You can take out `BAF_L85A2_RIS_SUSAT` and `BAF_L85A2_RIS_ACOG`, just remove them from the loot tables and add an update to 1.0.6.2 sql to swap them out for the closest equivalent.

File: SQF/dayz_code/external/R3F_Realism/R3F_Weight/Weapons/Rifles.hpp [hunk 2]
```diff
@@ -151,14 +151,106 @@ class AK_74_GL_kobra
...
+};
+class L85A2_CCO_SD_FL_DZ
+{
+	weight = 5.6;
+};
+class L85A2_CCO_SD_MFL_DZ
+{
+	weight = 5.6;
+};
 class L85_Holo_DZ
```
Comment (ebayShopper) [inline SQF/dayz_code/external/R3F_Realism/R3F_Weight/Weapons/Rifles.hpp]: This doesn't exist anymore, should be `L85A2_Holo_DZ`

**Comment by worldwidesorrow** [comment]
@ebayShopper How do I refresh my github repo. Do I just delete the one I have and re-fork it or switch the branches around and do a pull request? Sorry for being a noob. Do you still want me to add the changes you requested above?

**Comment by worldwidesorrow** [comment]
OK thanks.

**Pull request merged**